python-avd/pyavd/_eos_designs/structured_config/underlay/loopback_interfaces.py

```diff
@@ -31,19 +31,31 @@ def loopback_interfaces(self: AvdStructuredConfigUnderlay) -> list | None:
 
         loopback_interfaces = []
         # Loopback 0
-        loopback0 = {
-            "name": "Loopback0",
-            "description": self.shared_utils.interface_descriptions.router_id_loopback_interface(
-                InterfaceDescriptionData(
-                    shared_utils=self.shared_utils, interface="Loopback0", description=get(self._hostvars, "overlay_loopback_description")
+        if not self.shared_utils.underlay_ipv4:
```

Comment:
Let's just if/else for the ip_address vs. ipv6_address field instead of the whole block. Maybe it can be combined with some or case for the router_id thing in line 57.